### PR TITLE
Task: resume markup and styles

### PR DESCRIPTION
**Pull request by kasatka660**
The page contains resume created with html and css.
Link to it: https://kasatka660.github.io/rsschool-codejam1-cv/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+              <li><a href="https://github.com/kasatka660" target="_blank"><i class="fab fa-github"></i></a></li>
+              <li><a href="https://www.facebook.com/lena.khalyavskaya" target="_blank"><i class="fab fa-facebook-f" ></i></a></li>
+              <li><a href="https://vk.com/lenakhalyavskaya" target="_blank"><i class="fab fa-vk"></i></a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="container about-me" id="about">
```
Comment (OlgaMardvilko) [inline index.html:39]: Why did you register id here and below? I see that you did not use it for styling. This is not an error, just a little confusing to add attributes that are not used.

**Review by OlgaMardvilko — COMMENTED** [review]
Really great job and interesting resume! 
I liked how carefully and fascinatingly you approached the task.
Also, all the requirements for this task are met.

Score for this task *100*.

FYI: @manuminsk

File: index.html (hunk 1 above)
Comment (kasatka660) [reply index.html:39]: @OlgaMardvilko 
Thanks a lot for checking so fast and the feedback!

Ids of containers were used as anchors in navigation.
And the ones for skills blocks I forgot to remove unfortunately.

Comment (OlgaMardvilko) [reply index.html:39]: All is well!
You can make marge.

**Pull request merged**